app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -20,11 +20,11 @@
 
 .app-c-highlight-boxes__item {
   border: 1px solid $grey-2;
-  padding: $gutter-half $gutter-half $gutter;
+  padding: $gutter-half * 1.5;
   width: 100%;
```

Comment:
Is this a block level element? If so, width: 100% should be implied, so it doesn't need to be included.